_includes/footer.html

```diff
@@ -1,28 +1,57 @@
-<footer class="footer bg-navy">
-  <div class="footer-nav bg-lightest-blue">
+<footer class="footer">
+  <div class="footer-nav">
     <div class="container">
+      <!-- apply to .footer-nav -->
       <div class="navigation">
         <nav>
           <div class="grid">
+            <!-- apply to nav -->
             <section>
+              <!-- replace with uls -->
               <h6>{% t nav.groups.agencies %}</h6>
-              <a class="h6" target="_blank" href="https://partners.login.gov">{% t nav.become_a_partner %}</a>
-              <a class="h6" target="_blank" href="https://developers.login.gov">{% t nav.developer_guide %}</a>
+              <a
+                class="font-xs"
```

Comment:
`font-xs` was added to these links, but the font styles aren't actually being applied, since other styles exist which have higher specificity. It seems that these classes may as well be removed.

![image](https://user-images.githubusercontent.com/1779930/99004763-a05a3100-250d-11eb-83a4-dabee6e14bf4.png)